tf2/include/tf2/time.h

```diff
@@ -58,10 +58,9 @@ namespace tf2
 
   inline std::string displayTimePoint(const TimePoint& stamp)
   {
-    std::time_t time = std::chrono::system_clock::to_time_t(std::chrono::time_point_cast<std::chrono::milliseconds>(stamp));
-    char str[100];
-    std::strftime(str, sizeof(str), "%c", std::localtime(&time));
-    return std::string(str);
+    char buffer [100];
```

Comment:
It could be smaller, apparently depending on local etc 25 for a double known not to be long enough. I wanted to guarantee that we'd have enough space. I erred on the side of caution since bytes on the stack for a quick function call are relatively cheap. What do you suggest for the size of the buffer?  

I could also make it dynamically allocate the buffer to keep it to exactly the right size. https://stackoverflow.com/a/3923207/604099 but the extra complexity and computational cost of that seems overkill.